src/main/java/com/minecolonies/coremod/colony/managers/CitizenManager.java

```diff
@@ -100,9 +108,9 @@ public void writeToNBT(@NotNull final NBTTagCompound compound)
 
     @Override
     public void sendPackets(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this method to reduce its Cognitive Complexity from 23 to the 20 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)